common-rest/src/main/java/org/uniprot/api/rest/output/header/HttpCommonHeaderConfig.java

```diff
@@ -30,8 +30,10 @@
 @Configuration
 @Import({ServiceInfoConfig.class})
 public class HttpCommonHeaderConfig {
-    public static final String X_RELEASE = "X-Release";
+    public static final String X_RELEASE_NUMBER = "X-ReleaseNumber";
+    public static final String X_RELEASE_DATE = "X-ReleaseDate";
     static final String ALLOW_ALL_ORIGINS = "*";
+    public static final String X_TOTAL_RECORDS = "X-TotalRecords";
```

Comment:
maybe these should not be camelcase?

I've seen sometimes servers/traffic managers changing the headers to be lowercase (no idea why this happens sometimes).

so maybe they could be
x-release-number
x-release-date
x-total-records ?